addons/kube-ingress-aws-controller/README.md

```diff
@@ -88,12 +88,7 @@ kube-ingress-aws-controller, which we will use:
   "Action": [
     "acm:ListCertificates",
     "acm:DescribeCertificate",
-    "autoscaling:DescribeAutoScalingGroups",
-    "autoscaling:DescribeLoadBalancerTargetGroups",
-    "autoscaling:AttachLoadBalancers",
-    "autoscaling:DetachLoadBalancers",
-    "autoscaling:DetachLoadBalancerTargetGroups",
-    "autoscaling:AttachLoadBalancerTargetGroups",
+    "autoscaling:*",
```

Comment:
I wouldn't use `*` here, you just need these permissions: https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/senza-definition.yaml#L266-L271

`autoscaling:DescribeLoadBalancerTargetGroups` is missing.

Reply:
no it was no missing, see the second line